sdk/timeseriesinsights/Azure.IoT.TimeSeriesInsights/samples/TimeSeriesInsightsClientSample/TypesSamples.cs

```diff
@@ -0,0 +1,140 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using static Azure.IoT.TimeSeriesInsights.Samples.SampleLogger;
+
+namespace Azure.IoT.TimeSeriesInsights.Samples
+{
+    internal class TypesSamples
+    {
+        /// <summary>
+        /// This sample demonstrates usage of Time Series Insights types APIs.
+        /// </summary>
+        public async Task RunSamplesAsync(TimeSeriesInsightsClient client)
+        {
+            PrintHeader("TIME SERIES INSIGHTS TYPES SAMPLE");
+
+            // Get model settings
+            TimeSeriesModelSettings modelSettings = await client.ModelSettings.GetAsync().ConfigureAwait(false);
+
+            #region Snippet:TimeSeriesInsightsSampleCreateType
+            // Create an aggregate type
+            var timeSeriesTypes = new List<TimeSeriesType>();
+
+            var countExpression = new TimeSeriesExpression("count()");
+            var aggregateVariable = new AggregateVariable(countExpression);
+            var variables = new Dictionary<string, TimeSeriesVariable>();
+            var variableName = "aggregateVariable";
+            variables.Add(variableName, aggregateVariable);
+
+            var timeSeriesTypesProperties = new Dictionary<string, string>
+            {
+                { "Type1", Guid.NewGuid().ToString()},
+                { "Type2", Guid.NewGuid().ToString()}
+            };
+
+            foreach (KeyValuePair<string, string> property in timeSeriesTypesProperties)
+            {
+                var type = new TimeSeriesType(property.Key, variables)
+                {
+                    Id = property.Value
+                };
+                timeSeriesTypes.Add(type);
+            }
+
+            Response<TimeSeriesTypeOperationResult[]> createTypesResult = await client
+                .Types
+                .CreateOrReplaceAsync(timeSeriesTypes)
+                .ConfigureAwait(false);
+
+            // Check if the result error array does contain any object that is set
+            foreach (var result in createTypesResult.Value)
+            {
+                if (result.Error != null)
+                {
+                    Console.WriteLine($"Failed to create a Time Series Insights type: {result.Error.Message}.");
+                }
+            }
+            #endregion Snippet:TimeSeriesInsightsSampleCreateType
+
+            #region Snippet:TimeSeriesInsightsSampleGetTypeById
+            // Code snippet below shows getting a default Type using Id
+            Response<TimeSeriesTypeOperationResult[]> getTypeByIdResults = await client
```

Comment:
Maybe add a comment here clarifying what modelSettings is, because it's irrelevant in the scope of this region.
// The default type Id can be obtained programmatically by using the ModelSettings client.